odoo/api.py

```diff
@@ -709,6 +710,32 @@ def norecompute(self):
         """ Delay recomputations (deprecated: this is not the default behavior). """
         yield
 
+    @functools.lru_cache(maxsize=32)
```

Comment:
@odony 
Is not better using orm_cache?

Reply:
`ormcache` _is_ a LRU cache, and it works on model methods only

Reply:
Note that `functools.lru_cache` will cache the *function*, i.e., the pair `(self, field)` is used as a key in the cache dictionary.  Therefore, the cache is not specific to the environment, not even to the current request.

I suggest to implement an environment-specific cache for the method with a plain `dict`.  The cache size will never exceed the number of context-dependent fields in the corresponding registry.  When the current request is over, the environment becomes garbage, and its cache becomes garbage, too.